torchgeo/datamodules/inria.py

```diff
@@ -0,0 +1,244 @@
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License.
+
+"""InriaAerialImageLabeling datamodule."""
+
+import os
+from typing import Any, Dict, List, Optional, Tuple, Union, cast
+
+import kornia.augmentation as K
+import pytorch_lightning as pl
+import torch
+import torchvision.transforms as T
+from einops import rearrange
+from kornia.contrib import extract_tensor_patches
+from torch.nn.modules.utils import _pair
+from torch.utils.data import DataLoader, Dataset
+from torch.utils.data._utils.collate import default_collate
+
+from torchgeo.datamodules.utils import dataset_split
+from torchgeo.datasets import InriaAerialImageLabeling
+from torchgeo.datasets.utils import PredictDataset, compute_padding
+
+DEFAULT_AUGS = K.AugmentationSequential(
+    K.RandomHorizontalFlip(p=0.5),
+    K.RandomVerticalFlip(p=0.5),
+    data_keys=["input", "mask"],
+)
+
+
+def collate_wrapper(batch: List[Dict[str, Any]]) -> Dict[str, Any]:
+    """Flatten wrapper."""
+    r_batch: Dict[str, Any] = default_collate(batch)  # type: ignore[no-untyped-call]
+    r_batch["image"] = torch.flatten(r_batch["image"], 0, 1)
+    if "mask" in r_batch:
+        r_batch["mask"] = torch.flatten(r_batch["mask"], 0, 1)
+
+    return r_batch
+
+
+class InriaAerialImageLabelingDataModule(pl.LightningDataModule):
+    """LightningDataModule implementation for the InriaAerialImageLabeling dataset.
+
+    Uses the train/test splits from the dataset and further splits
+    the train split into train/val splits.
+
+    .. versionadded:: 0.3
+    """
+
+    h, w = 5000, 5000
+
+    def __init__(
+        self,
+        root_dir: str,
+        batch_size: int = 32,
+        num_workers: int = 0,
+        val_split_pct: float = 0.1,
+        test_split_pct: float = 0.1,
+        patch_size: Union[int, Tuple[int, int]] = 512,
+        num_patches_per_tile: int = 32,
+        augmentations: K.AugmentationSequential = DEFAULT_AUGS,
```

Comment:
This causes the following error while building docs

```
WARNING: py:class reference target not found: kornia.augmentation.container.augment.AugmentationSequential
```

Reply:
Can you try adding https://kornia.readthedocs.io/en/latest/ to `intersphinx_mapping` in `docs/conf.py`?

Reply:
That worked.